### ISSUE TITLE
Delete downloaded media from Xbox Live

### ISSUE THREAD

**Issue by jdaymude** [jdaymude/xbox2local#2]
**Description**
It may be the case that not all users want to have their screenshots and game clips remain on Xbox Live after they download them. A nice enhancement would be for xbox2local to optionally delete downloaded media from Xbox Live after successfully downloading it, e.g., with a new optional command line argument `--delete_after_downloading`.

However, neither [X API](https://xapi.us/documentation) nor the actual Xbox Live Services [RESTful API](https://docs.microsoft.com/en-us/gaming/xbox-live/api-ref/xbox-live-rest/atoc-xboxlivews-reference) seem to support this at the moment.

**Can you help develop and maintain this feature?**
Yes.

**Comment by jdaymude** [comment]
No longer on hold after #15, which updated the backend API to OpenXBL which contains an `/api/v2/dvr/gameclips/delete/{gameClipId}` endpoint for deleting gameclips.
